packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -554,38 +555,57 @@ const schema: SchemaType<"Posts"> = {
     graphQLtype: 'Int!',
     type: Number,
     canRead: ['guests'],
-    resolver: ({readTimeMinutesOverride, contents}: DbPost): number =>
-      Math.max(
-        1,
-        Math.round(typeof readTimeMinutesOverride === "number"
-          ? readTimeMinutesOverride
-          : (contents?.wordCount ?? 0) / READ_WORDS_PER_MINUTE)
-      ),
-    sqlResolver: ({field}) => `GREATEST(1, ROUND(COALESCE(
-      ${field("readTimeMinutesOverride")},
-      (${field("contents")}->'wordCount')::INTEGER
-    ) / ${READ_WORDS_PER_MINUTE}))`,
+    resolver: async (post: DbPost, _args: void, context: ResolverContext) => {
+      const normalizeValue = (value: number): number =>
```

Comment:
Probably prefer to define this elsewhere, since otherwise we're reallocating for it each time the field is resolved